src/superphot_plus/utils.py

```diff
@@ -154,13 +154,21 @@ def flux_model(cube, t_data, b_data):
         The time data.
     b_data : array-like
         The band data.
+    ordered_bands : array-like
+        The band names in the cube's parameter order.
+    ref_band : str
+        The base/reference band which all other values are multiples of.
 
     Returns
     -------
     f_model : numpy array
         The flux model for the given set of time and band data.
     """
-    A, beta, gamma, t0, tau_rise, tau_fall, es = cube[:7]  # pylint: disable=unused-variable
+    b_data = np.array(b_data)
+    ref_band_idx = np.argmax(ref_band == np.array(ordered_bands))
```

Comment:
The `argmax` threw me off a bit because you will only ever be expecting the a single match, right? I think this works. Alternatively you could use numpy's `where` clause, but I don't think it is necessary to switch it.

Reply:
Yeah, I couldn't think of an alternative using np.where, argwhere or np.nonzero that was necessarily shorter, so I'm sticking with this for now. I've confirmed it does typecast the bools into 1s and 0s when it evaluates the expression.